ledger-service/http-json/src/main/scala/com/digitalasset/http/CommandService.scala

```diff
@@ -150,18 +151,27 @@ class CommandService(
       jwtPayload: JwtWritePayload,
       meta: Option[domain.CommandMeta],
       command: lav1.commands.Command.Command,
+  )(implicit
+      lc: LoggingContextOf[InstanceUUID with RequestID]
   ): lav1.command_service.SubmitAndWaitRequest = {
-
     val commandId: lar.CommandId = meta.flatMap(_.commandId).getOrElse(uniqueCommandId())
-
-    Commands.submitAndWaitRequest(
-      jwtPayload.ledgerId,
-      jwtPayload.applicationId,
-      commandId,
-      jwtPayload.actAs,
-      jwtPayload.readAs,
-      command,
+    withEnrichedLoggingContext(
+      label[lar.CommandId],
+      Map("command_id" -> commandId.toString),
     )
+      .run { implicit lc =>
+        logger.info(
```

Comment:
We already log every request, e.g, `    logger.trace("sending exercise command to ledger")`.
I think what would be more helpful is to pass in the type of request (exercise, create, …) in here and then log something more specific than "doing a submit and wait request".